### DISCUSSION TITLE
試行版リリースに向けて行うこと・将来的に行うこと

### DISCUSSION THREAD

**Discussion by CLRafaelR** [CLRafaelR/lingdocdown#10] [General]
1. 試行版バージョン名: `Ain't no mountain high enough`
    - バージョン命名規則: Disco hitsまたは*future nostalgia*を感じる曲の名前
2. R・RStudioのインストール方法は，当座，1月30日ワークショップで使用したbeamerスライドで代用
    - [x] rtools40のインストール方法も入れること
    - [ ] 将来的には，テンプレート本体で内容をカバーする（1月30日ワークショップで使用したbeamerスライドは使わない）
3. 引用・参考文献の拡充
    - [x] テンプレート内でbibファイルを生成する
    - [x] 最低限以下をカバー
      - [x] `article`
      - [x] `inproceedings`
      - [ ] `manual`
        - [ ] Praat
        - [ ] R
      - [x] `book`: [An Introduction to Bayesian Data Analysis for Cognitive Science](https://vasishth.github.io/bayescogsci/book/)
      - [x] ~~`inbook`~~ `incollection`
      - [Bibliographies and Citations](https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#Citation_Styles)
    - [x] bibファイルでの入力の仕方
      - [x] ファーストネームを全部打ってよい（イニシャルにするかどうかはスタイルが自動で判断）
      - [x] 複数著者の場合，著者名同士の区切りはand
      - [x] 年号に括弧をつけてもつけなくてもよい
    - [ ] スタイルの変え方
    - [ ] cslが使えるか試す…？
 4. YAMLの拡充
     - [x] `geometry`（LSJの版面を設定）
     - [x] `keep_tex: TRUE`
     - [ ] `toc`
     - [x] `includes: in_header`
     - [x] `indent: TRUE`
     - [x] `documentclass`
     - [x] `classoption`
     - [ ] `reference_docx`: LSJテンプレート
       - [ ] http://www.ls-japan.org/modules/documents/LSJpapers/GengoKenkyu_template-Sep2020-JA.docx
       - [ ] http://www.ls-japan.org/modules/documents/LSJpapers/GengoKenkyu_template-Sep2020-EN.docx
     - [ ] `reference_doc`: スライド用

**Comment by CLRafaelR** [comment]
# 将来的に行うこと

- [x] bxjsarticleのclassoptionで`{ }`を使う場合に，`{ }`をエスケープさせない方法
  - ~~現状，下記はできていない~~できた！
```markdown
documentclass: bxjsarticle
classoption: 
  - pandoc 
  - |
    ```{=latex}
    japaram = {units = true}
    ```
```

**Reply by nakagawanatuko** [reply]
素晴らしいテンプレートを共有してくださってありがとうございます！

参考文献以外の『言語研究』のテンプレートを作ったことがありますので共有します。（1ページあたりの行数、1行あたりの文字数指定があるので大変つらかったです。）XeLaTeXで動きます。
需要があるかわかりませんが、自由に使ってください。
http://gofile.me/3k4gm/sFWku4e5W

**Reply by CLRafaelR** [reply]
ありがとうございます！行数と文字数指定がカバーされているのはいいですね！！

58行目以降の内容は，RMarkdownの段階（LaTeXが背後で動く前段階）から始まるため，そこを除いたファイルであれば，[`main.Rmd`](https://github.com/CLRafaelR/lingdocdown/blob/main/main.Rmd)の中に組み入れて使えます。`main.Rmd`のYAML sectionの`includes`以下を，次のように変更すればOKです。

```yaml
output: 
  bookdown::pdf_document2:
    latex_engine: xelatex
    keep_tex: TRUE
    includes:
      in_header:
        - your/path/to/GengoKenkyu.tex
```

**Reply by nakagawanatuko** [reply]
なるほど、そういう便利なこともできるんですね。
ありがとうございます。やってみます。
